compatibility/WORKSPACE

```
@@ -0,0 +1,113 @@
+workspace(name = "compatibility")
+
+# We import the main workspace for nix stuff and some shared Bazel rules.
+# This is mainly so we don’t have to symlink a bunch of files.
+# Note that you should never depend on targets from @daml.
+local_repository(
+    name = "daml",
+    path = "..",
+)
+load("//bazel_tools:os_info.bzl", "os_info")
```

Why not use `@daml` instead of a symlink like done below for the Nix files?
```suggestion
load("@daml//bazel_tools:os_info.bzl", "os_info")
```

---

Good point, I’ve switched to that. Thanks!